### DISCUSSION TITLE
Specifying the top of tg2 website

### DISCUSSION THREAD

**Discussion by mahdialibi** [TurboGears/tg2#135] [Q&A] [answered]
Hi, 
How can specify the top of tg2 website like that was done in tg1.x 
https://turbogears.org/1.0/docs/GettingStarted/URLs.html

I want to serve my website as : https://web.site/suburl
suburl must be configured
thank you

**Comment by amol-** [comment, marked as answer]
You can use `SCRIPT_NAME` to specify the mountpoint of a web app. 
The script name should usually be set somewhere in your WSGI server.

It's also mentioned in https://turbogears.readthedocs.io/en/latest/turbogears/widgets_forms.html#relocatable-widget-actions in the context of relocatable widgets, but it's generally how it works with WSGI
